server/core/src/main/java/org/infinispan/server/core/transport/NettyTransport.java

```diff
@@ -185,7 +190,7 @@ public String getHostName() {
 
    @Override
    public String getPort() {
-      return Integer.toString(address.getPort());
+      return nettyPort.orElse(address.getPort()).toString();
```

Comment:
I assumed I can not do it, since this is exposed by JMX. But digging further I think JMX utils will handle this... changing to `int`.